adafruit_display_text/label.py

```diff
@@ -292,7 +292,8 @@ def _update_text(
         self._text = new_text
         self._boundingbox = (left, top, left + right, bottom - top)
 
-        self._update_background_color(self._background_color)
+        if self.background_color:
```

Comment:
Can background color be 0x000000 for black? That will fail this test. I think you want:
```suggestion
        if self.background_color is not None:
```

(I just ran into this with requests and a page of 0 content length.)

Reply:
Whoops. Thank you pointing that out. The new commit changes to use `is not None`